### PR TITLE
docs: add facebook connector template

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

docs: add facebook connector template

### CODE REVIEW

File: tutorial/build-with-logto/_connector-facebook.mdx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+2. Click your existing app or [create a new one](https://developers.facebook.com/docs/development/create-an-app) if needed.
+   - The selected [app type](https://developers.facebook.com/docs/development/create-an-app/app-dashboard/app-types) is up to you, but it should have the product _Facebook Login_.
+3. On the app dashboard page, scroll to the "Add a product" section and click the "Set up" button on the "Facebook Login" card.
+4. Skip the Facebook Login Quickstart page, and click the sidebar -> "Products" -> "Facebook Login" -> "Settings".
+5. In the Facebook Login Settings page, fill `${your_logto_origin}/callback/${connector_id}` in the "Valid OAuth Redirect URIs" field. The `connector_id` can be found on the top bar of the Logto Admin Console connector details page. E.g.:
+   - `https://logto.dev/callback/${connector_id}` for production
+   - `https://localhost:3001/callback/${connector_id}` for testing in the local environment
+6. Click the "Save changes" button at the bottom right corner.
+
+### Compose the connector JSON
```
Comment (wangsijie) [inline tutorial/build-with-logto/_connector-facebook.mdx:17]: need to update? we are using form now

Comment (xiaoyijun) [reply tutorial/build-with-logto/_connector-facebook.mdx]: updated.

**Review by darcyYe — APPROVED** [review]

**Review by wangsijie — APPROVED** [review]

File: tutorial/build-with-logto/_connector-facebook.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+| Name         | Type   |
+| ------------ | ------ |
+| clientId     | string |
+| clientSecret | string |
+
+#### Facebook developer docs
+
+- [Facebook Login - Documentation - Facebook for Developers](https://developers.facebook.com/docs/facebook-login/)
+  - [Manually Build a Login Flow](https://developers.facebook.com/docs/facebook-login/guides/advanced/manual-flow/)
```
Comment (charIeszhao) [inline tutorial/build-with-logto/_connector-facebook.mdx:52]: Unnecessary spaces before the `-`?

File: tutorial/build-with-logto/_connector-facebook.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+- In the app dashboard page, click the sidebar -> "Roles" -> "Test Users".
+- Click the "Create test users" button to create a testing user.
+- Click the "Options" button of the existing test user, and you will see more operations, e.g., "Change name and password".
+
+### Publish Facebook sign-in settings
+
+Usually, only the test, admin, and developer users can sign in with the related app under [development mode](https://developers.facebook.com/docs/development/build-and-test/app-modes#development-mode).
+
+To enable normal Facebook users sign-in with the app in the production environment, you maybe need to switch your Facebook app to _[live mode](https://developers.facebook.com/docs/development/build-and-test/app-modes#live-mode)_, depending on the app type.
+E.g., the pure _business type_ app doesn't have the "live" switch button, but it won't block your use.
```
Comment (charIeszhao) [inline tutorial/build-with-logto/_connector-facebook.mdx:33]: `block your use` sounds a bit strange, how about `block you from using it` or something better?

File: tutorial/build-with-logto/_connector-facebook.mdx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+   - `https://logto.dev/callback/${connector_id}` for production
+   - `https://localhost:3001/callback/${connector_id}` for testing in the local environment
+6. Click the "Save changes" button at the bottom right corner.
+7. In the Facebook app dashboard page, click the sidebar -> "Settings" -> "Basic" and then you will get the **App ID** and **App secret**. The _App ID_ is the `clientId` in your Logto connector conifg and the _App secret_ is the `clientSecret`.
+
+### Test sign-in with Facebook's test users
+
+You can use the accounts of the test, developer, and admin users to test sign-in with the related app under both development and live [app modes](https://developers.facebook.com/docs/development/build-and-test/app-modes).
+
+You can also "take the app live" directly so that any Facebook user can sign in with the app.
```
Comment (charIeszhao) [inline tutorial/build-with-logto/_connector-facebook.mdx:22]: What does it mean by `take the app live`?

File: tutorial/build-with-logto/_connector-facebook.mdx (hunk 2 above)
Comment (xiaoyijun) [reply tutorial/build-with-logto/_connector-facebook.mdx]: Updated.

File: tutorial/build-with-logto/_connector-facebook.mdx (hunk 3 above)
Comment (xiaoyijun) [reply tutorial/build-with-logto/_connector-facebook.mdx]: Updated.

File: tutorial/build-with-logto/_connector-facebook.mdx (hunk 4 above)
Comment (xiaoyijun) [reply tutorial/build-with-logto/_connector-facebook.mdx]: This should be:
set the app to "live mode" by "switching mode"
Add related link is added.

File: tutorial/build-with-logto/_connector-facebook.mdx [hunk 5]
```diff
@@ -0,0 +1,53 @@
+### Register a Facebook developer account
+
+[Register as a Facebook Developer](https://developers.facebook.com/docs/development/register/) if you don't already have one.
```
Comment (charIeszhao) [inline tutorial/build-with-logto/_connector-facebook.mdx:3]:
```suggestion
[Register as a Facebook Developer](https://developers.facebook.com/docs/development/register/) if you don't have one.
```

File: tutorial/build-with-logto/_connector-facebook.mdx [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+### Set up a Facebook app
+
+1. Visit the [Apps](https://developers.facebook.com/apps) page.
+2. Click your existing app or [create a new one](https://developers.facebook.com/docs/development/create-an-app) if needed.
+   - The selected [app type](https://developers.facebook.com/docs/development/create-an-app/app-dashboard/app-types) is up to you, but it should have the product _Facebook Login_.
+3. On the app dashboard page, scroll to the "Add a product" section and click the "Set up" button on the "Facebook Login" card.
+4. Skip the Facebook Login Quickstart page, and click the sidebar -> "Products" -> "Facebook Login" -> "Settings".
+5. In the Facebook Login Settings page, fill `${your_logto_origin}/callback/${connector_id}` in the "Valid OAuth Redirect URIs" field. The `connector_id` can be found on the top bar of the Logto Admin Console connector details page. E.g.:
+   - `https://logto.dev/callback/${connector_id}` for production
```
Comment (charIeszhao) [inline tutorial/build-with-logto/_connector-facebook.mdx:13]: Change `logto.dev` to something generic?

File: tutorial/build-with-logto/_connector-facebook.mdx [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+- In the app dashboard page, click the sidebar -> "Roles" -> "Test Users".
+- Click the "Create test users" button to create a testing user.
+- Click the "Options" button of the existing test user, and you will see more operations, e.g., "Change name and password".
+
+### Publish Facebook sign-in settings
+
+Usually, only the test, admin, and developer users can sign in with the related app under [development mode](https://developers.facebook.com/docs/development/build-and-test/app-modes#development-mode).
+
+To enable normal Facebook users sign-in with the app in the production environment, you maybe need to switch your Facebook app to _[live mode](https://developers.facebook.com/docs/development/build-and-test/app-modes#live-mode)_, depending on the app type.
```
Comment (charIeszhao) [inline tutorial/build-with-logto/_connector-facebook.mdx:32]:
```suggestion
To enable normal Facebook users sign-in with the app in the production environment, you may need to switch your Facebook app to _[live mode](https://developers.facebook.com/docs/development/build-and-test/app-modes#live-mode)_, depending on the app type.
```

Comment (charIeszhao) [reply tutorial/build-with-logto/_connector-facebook.mdx:32]: Also what is `depending on the app type`? Does it mean if the app type is something else, then the user may not have to switch the facebook app to live mode? Why?

File: tutorial/build-with-logto/_connector-facebook.mdx (hunk 6 above)
Comment (xiaoyijun) [reply tutorial/build-with-logto/_connector-facebook.mdx:13]: This is an example.

Comment (xiaoyijun) [reply tutorial/build-with-logto/_connector-facebook.mdx]: Updated.

**Review by charIeszhao — APPROVED** [review]

**Pull request merged**